### DISCUSSION TITLE
Add a comment that this value will be updated when a new file is loaded

### DISCUSSION THREAD

**Discussion by Harding-Stardust** [arizvisa/ida-minsc#185] [Ideas]
It's not obvious that this value will be updated when a new file is loaded, please add a comment to clarify this.

https://github.com/arizvisa/ida-minsc/blob/af07f167a0992aed3f3a2a1670593880baa57820/base/database.py#L69C8-L69C8

**Comment by arizvisa** [comment]
converted to discussion.

**Comment by arizvisa** [comment]
Generally "missing comments" are not an issue, so I'm not quite sure how to respond to this. To elaborate, it comes across as you saying: "hey man, i can't read your free book. can you rewrite your free book so that i can read it easier?" Apologies if this sounds harsh, but when it comes to software, especially open-source software, it is not any maintainer's job to hold your hand through reading what they provide for you. 

Code is meant to be executed, and as much people want to believe that all code is art, the fact of the matter is that all code is speech and the intended listener is not humans, but rather the platforms that execute them. It's clearly documentation that is written specifically for humans to consume. That's not to say that humans can't read code, but it's not the goal of writing it and there's no reason that it should be.

To focus on the topic, both comments and documentation do not have an effect on anything but the small handful of people who're reading code. Therefore a more effective way to spend engineering time would be to improve the actual capabilities of the software itself. So, I suggest that if you really want great comments in code for humans to read, then consider contributing a PR. In all actuality, the documentation building system could use a major rewrite and better documentation.

**Comment by Harding-Stardust** [comment]
> To elaborate, it comes across as you saying: "hey man, i can't read your free book. can you rewrite your free book so that i can read it easier?" Apologies if this sounds harsh, but when it comes to software, especially open-source software, it is not any maintainer's job to hold your hand through reading what they provide for you.

I apologize if it sounds like that, I just wanted to help any other reader of the code that this value is not stale and will be updated with the notify_when triggers.
